src/components/Card/Card.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+interface CardProps {
+    /** Uh id  */
+    id?: string;
+
+    /** ClassName that appears in addition to "card" */
+    extraClassName?: string[];
```

Comment:
Can we simply call this `className`? It think it's assumed that it'll be added to the existing styles

Reply:
I think I had just mistakenly assumed it'd be a protected variable, but it looks like that isn't the case so sure :)